src/nasem_dairy/NASEM_equations/dev_milk_equations.py

```diff
@@ -37,3 +37,144 @@ def calculate_Mlk_CP_g(Mlk_NP_g):
     Mlk_CP_g = Mlk_NP_g / 0.95  # Line 2213
     return Mlk_CP_g
 
+
+def calculate_An_LactDay_MlkPred(An_LactDay: int) -> int:
+    """
+    An_LactDay_MlkPred: An_LactDay but capped at day 375
+    """
+    # Cap DIM at 375 d to prevent the polynomial from getting out of range, Line 2259
+    if An_LactDay <= 375:
+        An_LactDay_MlkPred = An_LactDay
+    elif An_LactDay > 375:
+        An_LactDay_MlkPred = 375
+
+    return An_LactDay_MlkPred
+
+
+def calculate_Trg_Mlk_Fat(Trg_MilkProd: float, Trg_MilkFatp: float) -> float:
+    """
+    Trg_Mlk_Fat: Target milk fat in kg, based on user input milk production and fat % 
+    """
+    Trg_Mlk_Fat = Trg_MilkProd * Trg_MilkFatp / 100  # Line 2262
+    return Trg_Mlk_Fat
+
+
+def calculate_Trg_Mlk_Fat_g(Trg_Mlk_Fat: float) -> float:
+    """
+    Trg_Mlk_Fat_g: Target milk fat g, based on user input milk production and fat % 
+    """
+    Trg_Mlk_Fat_g = Trg_Mlk_Fat * 1000  # Line 2263
+    return Trg_Mlk_Fat_g
+
+
+def calculate_Mlk_Fatemp_g(An_StatePhys: str, An_LactDay_MlkPred: int, Dt_DMIn: float, Dt_FAIn: float, Dt_DigC160In: float, Dt_DigC183In: float, Abs_Ile_g: float, Abs_Met_g: float):
+    """
+    Mlk_Fatemp_g: Milk fat prediciton, g, from Daley et al. no year given 
+    """
```

Comment:
Have copied some abbreviations across from old code. Can you add the missing ones please

Reply:
Added the 3 missing abbreviations